docs/docfx.json

```diff
@@ -0,0 +1,71 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "src": "../",
+          "files": [
+            "src/**/bin/**/**/Starnight.dll",
+            "src/**/bin/**/**/Starnight.Caching.dll",
+            "src/**/bin/**/**/Starnight.Caching.Providers.dll",
+            "src/**/bin/**/**/Starnight.Shared.dll",
+            "gen/bin/**/**/Starnight.SourceGenerators.dll"
```

Comment:
do we really want to generate docs for the source generators? also, issue or not, please do include Starnight.Internal for now

Reply:
gonna add Starnight.Internal, not sure about sgens